### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by vfdev-5**
Added 2 templates:
- Bug/Questions/Feature Requests
- User Feedback

Added a section in the readme about user feedback form.

@justusschock what do you think ?

### CODE REVIEW

**Comment by justusschock** [comment]
I think, we should have a separate template for bugs, where the user is also asked for his environment (os, python version, ignite version, how did he install [source, pip, conda], pytorch version). This would make it easier to track down version specific bugs.

Otherwise I like the new templates

**Comment by anmolsjoshi** [comment]
I agree with @justusschock, including a space to add stacktrace and versions will help people search past issues better and the team reproduce better 

PyTorch’s repo has good issue templates! Maybe we can borrow from there?

**Comment by vfdev-5** [comment]
@justusschock @anmolsjoshi thanks for the feedback! 

Main idea for was to introduce a split between issue/question/FR and a free form user feedback.
Personnally, I do not like much prefilled forms for issues as they looks a bit restrictive and never know how to fill it correctly. If we can found out something friendly.

**Comment by vfdev-5** [comment]
@justusschock @anmolsjoshi I created 3 templates for bug, FR, questions instead of 1. Please, take a look and we can go on. Thanks

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+name: "\U0001F41B Bug Report"
+about: Submit a bug report to help us improve Ignite
+
```
Comment (justusschock) [inline .github/ISSUE_TEMPLATE/bug-report.md]: Can we also include a label here?

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+name: "\U0001F680 Feature Request"
+about: Submit a proposal/request for a new Ingite feature
+
```
Comment (justusschock) [inline .github/ISSUE_TEMPLATE/feature-request.md]: Can we also include a label like `enhancement` here?

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+## 🐛 Bug description
+
+<!-- A clear and concise description of what the bug is. -->
+<!-- Please, add steps on how to reproduce it. -->
+<!-- If you have a code sample, error messages, stack traces, please provide it here as well -->
+<!-- A clear and concise description of what you expected to happen. -->
+
+## Environment
+
+Please copy and paste the output from
```
[the remark this answers is not in the document]
Comment (vfdev-5) [reply .github/ISSUE_TEMPLATE/bug-report.md]: This automates things :)
but I agree that main questions are asked below.

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (vfdev-5) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Well, I reported bug may not be ignite bug :)

File: .github/ISSUE_TEMPLATE/feature-request.md (hunk 2 above)
Comment (vfdev-5) [reply .github/ISSUE_TEMPLATE/feature-request.md]: I think we need to approve that a proposed FR is an enhancement.

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (vfdev-5) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Removed script part to simplify things

**Pull request merged**